finalPrep/annotations.js

```diff
@@ -2,41 +2,41 @@
 // Use the double // to create a new comment
 
 //Build a Bear
-function buildABear(name, age, fur, clothes, specialPower) {
-  var greeting = `Hey partner! My name is ${name} - will you be my friend?!`;
-  var demographics = [name, age];
-  var powerSaying = "Did you know that I can " + specialPower + " ?";
-  var builtBear = {
-    basicInfo: demographics,
-    clothes: clothes,
-    exterior: fur,
-    cost: 49.99,
-    sayings: [greeting, powerSaying, "Goodnight my friend!"],
-    isCuddly: true,
+function buildABear(name, age, fur, clothes, specialPower) { //creating function with 5 parameters
+  var greeting = `Hey partner! My name is ${name} - will you be my friend?!`; //created string with interpolation
+  var demographics = [name, age]; //created array with two parameters
+  var powerSaying = "Did you know that I can " + specialPower + " ?";//creating string with concatenation
+  var builtBear = { //creating var that's an object that compiles parameters and variables
+    basicInfo: demographics,//key with var demographics as value
+    clothes: clothes,//key with parameter as value
+    exterior: fur,//key with parameter as value
+    cost: 49.99,//key with interger as value
+    sayings: [greeting, powerSaying, "Goodnight my friend!"],//key with two variables and a string as a value
+    isCuddly: true,//key with boolean value
   };
 
-  return builtBear
+  return builtBear //returing buildBear Object
 }
 
-buildABear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares');
-buildABear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in');
+buildABear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares');//object instance for buildABear
+buildABear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in');//object instance for buildABear
```

Comment:
I would be specific here- you have the result of what happens with this line of code, but what is really being done is that the `buildABear` function is called.